core/src/main/java/io/questdb/griffin/model/QueryModel.java

```diff
@@ -646,7 +659,8 @@ public boolean equals(Object o) {
                 && Objects.equals(unionModel, that.unionModel)
                 && Objects.equals(updateTableModel, that.updateTableModel)
                 && Objects.equals(updateTableToken, that.updateTableToken)
-                && skipped == that.skipped;
+                && skipped == that.skipped
+                && allowPropagationOfOrderByAdvice == that.allowPropagationOfOrderByAdvice;
```

Comment:
Could you modify `QueryModelTest` to verify this check, as well as the `clear()` method modification?